test/functional/feature_maxtipage.py

```diff
@@ -43,6 +43,10 @@ def test_maxtipage(self, maxtipage, set_parameter=True, test_deltas=True):
         self.generate(node_miner, 1)
         assert_equal(node_ibd.getblockchaininfo()['initialblockdownload'], False)
 
+        # reset time to system time so we don't have a time offset with the ibd node the next
+        # time we connect to it, ensuring TimeOffsets::WarnIfOutOfSync() doesn't output to stderr
+        node_miner.setmocktime(0)
```

Comment:
Could squash commits:
587349072f `[net processing] Use TimeOffsets to warn when node is out of sync`
d790e776ca `[test] reset miner mocktime in maxtipage to prevent time offset warning`